apps/opik-backend/src/main/java/com/comet/opik/api/DatasetItemBatch.java

```diff
@@ -27,7 +27,8 @@ public record DatasetItemBatch(
                 DatasetItem.View.Write.class}) @Pattern(regexp = NULL_OR_NOT_BLANK, message = "must not be blank") @Schema(description = "If null, dataset_id must be provided") String datasetName,
         @JsonView({
                 DatasetItem.View.Write.class}) @Schema(description = "If null, dataset_name must be provided") UUID datasetId,
-        @JsonView({DatasetItem.View.Write.class}) @NotNull @Size(min = 1, max = 1000) @Valid List<DatasetItem> items)
+        @JsonView({DatasetItem.View.Write.class}) @NotNull @Size(min = 1, max = 1000) @Valid List<DatasetItem> items,
+        @JsonView({DatasetItem.View.Write.class}) @Schema(description = "Whether to include trace metadata (tags, comments, feedback scores, etc.) in dataset items", defaultValue = "false") Boolean includeTraceMetadata)
```

Comment:
Do you think it makes sense? Feedback scores cannot be associated with dataset items or comments as well.